### ISSUE TITLE
[Discussion] How should we do voting?

### ISSUE THREAD

**Issue by jogerj** [tl-masterlist/VTuber-Translators-Master-List#4]
Options:

- Divide by "Great", "Decent", "Bad", and "Recommended Content Creator" (as per before)
- Just vote "good" or "bad" and place them accordingly later
- Vote "Great", "Decent", or "Bad" for TLs. Content creators are voted separately 
- @vaendryl suggested using the Steam review formula
![Steam formula](https://cdn.discordapp.com/attachments/681824255922536565/760366342876037140/formula.png)
![Steam conversion chart](https://cdn.discordapp.com/attachments/681824255922536565/760367547282489364/p5etwl9mo5o51.png)
- Suggest your own in comments

**Comment by minamisoro** [comment]
since I think the general vetting process is similar to beatmap ranking process of game called osu, we should steal their workflow because they've been working on this system for many years and polished it over time:
1. **regular users vote on translators.** Translators with high votes get priority screening because it indicates high interest. However, TLC isn't really confined to highly voted translators and is free to choose who to review.
2. the translators will get "nominated" once 2 TLC checks their translations and they deem that quality is good. In this phase, another TLC can vote against other's nomination when they think that the translator is not fit for the list. **In other words, yes vote has to be +2 over the no vote**
3. **the nominated translators will sit in "qualified" status for x days**. In this phase, any TLCs can look at the nominated translators and check their work. **When there's any problems, TLC can veto that nomination** and the translator has to repeat the process again. The veto should be done sparingly, and they must provide a very convincing reasoning behind the veto
4. **once the translator survives x days of screening, translator will be added to the list**

tldr: TLC vote should be no + 2 for yes, make "qualified" status for screening, and add another voting system so regular people can have some input on who gets checked first  
link to the actual rule referenced for more detail: https://osu.ppy.sh/help/wiki/Beatmap_ranking_procedure

few things:
* "decent" vs "great" translators - just an idea but maybe during the nomination process, the TLC can choose to nominate the translator as "decent" and "great", and when both TLC decide to nominate them as "great", it can require additional 1 yes vote to move to qualified status?
* there should be people dedicated to check qualification, like a "TLC checker" that guarantees there have been some checks done to the "qualified" translators and act like a final gate keeper to the list
